powershap/shap_wrappers/shap_explainer.py

```diff
@@ -172,7 +171,8 @@ def explain(
             Shap_values = np.abs(Shap_values)
 
             if len(np.shape(Shap_values)) > 2:
-                Shap_values = np.max(Shap_values, axis=0)
+                # Shap_values = np.max(Shap_values, axis=0)
+                Shap_values = np.max(Shap_values, axis=0).T
```

Comment:
Can someone verify this change, for me the `test_ensemble_class_powershap` test function fails if I do not perform this transpose.